### PR TITLE
added additional tests

### PR DESCRIPTION
**Pull request by bluec0re**
radare/radare2#5898

### CODE REVIEW

**Comment by Maijin** [comment]
If they are currently broken, add BROKEN=1

**Comment by Maijin** [comment]
otherwise we can't merge

**Comment by radare** [comment]
PING

**Comment by bluec0re** [comment]
The one which is broken is already marked as broken. The other is just for completion (at least at the time I created the test). I've currently no access to a computer, therefore can't retest.

File: t/cmd_project [hunk 1]
```diff
@@ -185,3 +185,55 @@ EXPECT='fV 49 0x400410
...
+tk char **=type
+tk type.char **=**z
+tk type.char **.size=64
+Ps custom_types > /dev/null
+Pc custom_types~?char **
+"td struct Foo { int bar; int x, int y }"
+Ps custom_types > /dev/null
+Pc custom_types~?char **
+Pc custom_types~?Foo
+rm /tmp/custom_types
```
Comment (radare) [inline t/cmd_project]: dont use /tmp

File: t/cmd_project [hunk 2]
```diff
@@ -185,3 +185,55 @@ EXPECT='fV 49 0x400410
 fV 50 0x400412
 '
 run_test
+
+NAME='Custom types in project file'
+FILE=../bins/elf/analysis/main
+BROKEN=
+ARGS=
+CMDS='e dir.projects = /tmp/
```
Comment (radare) [inline t/cmd_project]: dont use tmp

**Comment by bluec0re** [comment]
Will change that. I've copied the tmp thing from other tests.

File: t/cmd_project [hunk 3]
```diff
@@ -200,9 +199,7 @@ Pc custom_types~?char **
 Ps custom_types > /dev/null
 Pc custom_types~?char **
 Pc custom_types~?Foo
-rm /tmp/custom_types
-rm /tmp/custom_types.d/xref
-rm /tmp/custom_types.d
+Pd custom_types > /dev/null
```
Comment (radare) [inline t/cmd_project]: maybe use e scr.null=true instead of piping to a system device that may not exist on non-unixes

File: t/cmd_project [hunk 4]
```diff
@@ -227,9 +223,7 @@ tk~?Foo
 Po custom_types > /dev/null
 tk~?char **
 tk~?Foo
-rm /tmp/custom_types
-rm /tmp/custom_types.d/xref
-rm /tmp/custom_types.d
+Pd custom_types > /dev/null
```
Comment (radare) [inline t/cmd_project]: same here

File: t/cmd_project (hunk 3 above)
Comment (bluec0re) [inline t/cmd_project]: Doesn't work. ``e scr.null=true`` leads to an empty project file:

```
# r2 rdb project file
# flags
# eval
# sections
# meta
# types
# macros
```

**Comment by radare** [comment]
in fact ive seen that there are other tests that are using /tmp too :(((

**Pull request merged**